pkg/cluster/spec/spec.go

```diff
@@ -58,7 +58,7 @@ type (
 	// specification in topology.yaml
 	GlobalOptions struct {
 		User            string               `yaml:"user,omitempty" default:"tidb"`
-		Group           string               `yaml:"group,omitempty"`
+		Group           string               `yaml:"group,omitempty" default:"pingcap"`
```

Comment:
It's not a good idea to set a default value for `Group`, because by default, it should be same with `User`. 
https://github.com/pingcap/tiup/blob/master/pkg/cluster/module/user.go#L86

Reply:
OK, I'll revert it. I thought this was a forgotten place.